ui/App/DiagnosticsScreen/Storage.svelte

```diff
@@ -0,0 +1,40 @@
+<!--
+ Copyright © 2021 The Radicle Upstream Contributors
+
+ This file is part of radicle-upstream, distributed under the GPLv3
+ with Radicle Linking Exception. For full terms see the included
+ LICENSE file.
+-->
+<script lang="ts">
+  import * as proxy from "ui/src/proxy";
+</script>
+
+<style>
+  pre {
+    overflow: scroll;
+    font-family: var(--typeface-mono-regular);
+    font-size: 14px;
+  }
+
+  .section {
+    margin-bottom: 1rem;
+  }
+
+  .state {
+    background-color: var(--color-foreground-level-1);
+    border-radius: 0.5rem;
+    padding: 1rem;
+    overflow: scroll;
+  }
+</style>
+
+<div class="section">
+  {#await proxy.client.diagnosticsGet()}
+    Loading…
```

Comment:
Does it make sense to use the loading components?

Reply:
The request is probably quick enough.

94cf1828ed9ef599b9ba55691ef94fc3a82a7aea